package.json

```diff
@@ -25,26 +29,26 @@
     "ethereumjs-testrpc-sc": "6.0.7",
     "istanbul": "^0.4.5",
     "keccakjs": "^0.2.1",
-    "req-cwd": "^1.0.1",
+    "req-cwd": "^2.0.0",
     "shelljs": "^0.7.4",
     "sol-explore": "^1.6.2",
-    "solidity-parser-sc": "0.4.1",
+    "solparse": "^1.4.0",
```

Comment:
`solparse` is great - it's a more robust implementation than `sc` and is very well maintained. And we have a friendly collaboration with them because we both helped with the upstream consensys parser. 

That said, there are benefits to self-publishing `sc` - for example it requires less coordination and we can often get away with much coarser parsing than they can - they're supporting a linter. If this was painful to maintain I would `solparse` in a second but atm it's not and it's convenient to be able to quickly add the occasional over-permissive rule to stop `solidity-coverage` from crashing. 

Also we're actively trying to re-design ourselves around the `solc` ast in #111 and #150. 

Open to persuasion here but I'd rather not do this in this PR.

Reply:
`solidity-parser-sc` is terribly out of date dependency-wise vs `solparser` (which I recently contributed some changes to).  I think it makes sense to consolidate dependencies wherever possible.  The alternative is fixing all the out of date dependencies in `solidity-parser-sc` as well and effectively keeping them in sync.  thoughts?

Reply:
Eminently reasonable. In fact we went through a long period of trying to unify under a single parser and `solparse` is definitely the best thing out there right now. Thanks for contributing to it - fantastic. 

On reflection, I'd rather preserve the flexibility/control that self-publishing provides since we're trying to move to a different AST and it's important to keep our parsing super consistent during that process so we know what behavior we're duplicating. 

This change can't go in this PR, I'm really sorry. It's too consequential. I think it's ok for the time being if `sc` has out-of-date deps, and we should revisit using `solparse` if it looks like we'll be stuck in the current design for a while or the parser breaks.

Reply:
@cgewecke where is the repo for `solidity-parser-sc`?  If you are really wedded to that then I'd like to fix the dependencies.